### PR TITLE
feat(cli.admin.helpers): users can now interactively select keys on disk

### PR DESCRIPTION
**Pull request by trishankatdatadog**
# Description

Users can now interactively select keys as files on disk instead of manually typing or at least pasting their entire paths.

Fixes #(issue): N/A
# Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality not to work as expected)

# Additional requirements
- [ ] Tests have been added for the bug fix or new feature
- [ ] Docs have been added for the bug fix or new feature

# Code of Conduct
By submitting this PR, you agree to follow our [Code of Conduct](CODE_OF_CONDUCT.rst).
- [x] I agree to follow this project's Code of Conduct

### CODE REVIEW

**Comment by kairoaraujo** [comment]
Hi @trishankatdatadog, It has very cool functionality! 🎉 

To be able to merge it, we will need some extra work (sorry 😃 ):

- We need to add the new dependency as required during installation 
   https://github.com/repository-service-tuf/repository-service-tuf-cli/blob/c69a35e58b267d40499918a35407fcda37001e24/pyproject.toml#L25-L26
   
- We need to work on the unit tests and also the functional tests. Mostly of our Unit Tests and Functional Tests uses "user iteration" in mocked way. 

   - Unit tests are in tests/unit/
   - Functional Tests are in the umbrella repository

   I can help you with them

**Comment by trishankatdatadog** [comment]
> We need to work on the unit tests and also the functional tests. Mostly of our Unit Tests and Functional Tests uses "user iteration" in mocked way.

What do you need the tests to do?

**Comment by trishankatdatadog** [comment]
> What do you need the tests to do?

Nvm, I understood day before yesterday after some investigation: `prompt_toolkit.prompt()` needs [patching](https://python-prompt-toolkit.readthedocs.io/en/stable/pages/advanced_topics/unit_testing.html) somehow.

**Comment by trishankatdatadog** [comment]
@kairoaraujo sorry, don't understand: could you tell me why 905a1d5780f932490ca333e424a0737a592f9afd doesn't work?

File: tests/unit/cli/admin/test_ceremony.py [hunk 1]
```diff
@@ -312,11 +347,20 @@ def test_ceremony_online_key_one_of_root_keys(
...
-            f"{_PEMS / '0d9d3d4bad91c455bc03921daa95774576b86625ac45570d0cac025b08e65043.pub'}",  # Please enter path to public key  # noqa
             "Online Key",  # Please enter a key name
         ]
 
+        selection = iter(
+            (
+                f"{_PEMS / 'JH.pub'}",  # Please enter path to public key
+                f"{_PEMS / '0d9d3d4bad91c455bc03921daa95774576b86625ac45570d0cac025b08e65043.pub'}",  # Please enter path to public key  # noqa
+            )
+        )
```
Comment (kairoaraujo) [inline tests/unit/cli/admin/test_ceremony.py:358]: The problem is that you need the first prompt_toolkit to input the root keys. You mocked only the iterations for the online key.

```suggestion
        selection = iter(
            (
                f"{_PEMS / 'JC.pub'}",   # Root key 1
                f"{_PEMS / 'JH.pub'}",   # Root key 2
                f"{_PEMS / 'JJ.pub'}",    # Root key 3           
                f"{_PEMS / 'JH.pub'}",  # Please enter path to public key
                f"{_PEMS / '0d9d3d4bad91c455bc03921daa95774576b86625ac45570d0cac025b08e65043.pub'}",  # Please enter path to public key  # noqa
            )
        )
```

**Comment by kairoaraujo** [comment]
> @kairoaraujo sorry, don't understand: could you tell me why [905a1d5](https://github.com/repository-service-tuf/repository-service-tuf-cli/commit/905a1d5780f932490ca333e424a0737a592f9afd) doesn't work?

See https://github.com/repository-service-tuf/repository-service-tuf-cli/pull/712/files#r1796535461

Basically when you mock, you need to add all the iterations that will call prompt_toolkit

File: tests/unit/cli/admin/test_ceremony.py (hunk 1 above)
Comment (trishankatdatadog) [reply tests/unit/cli/admin/test_ceremony.py:358]: But where are the inputs to those keys coming from? It's not obvious to me.

**Comment by trishankatdatadog** [comment]
@kairoaraujo Sorry it took a while, but:
1. How do you want to address [coverage](https://github.com/repository-service-tuf/repository-service-tuf-cli/pull/712/files#annotation_27642788003)?
2. I didn't want to repeat the monkeypatches, but I couldn't figure out how not to DRY here.
3. Would you like to use `prompt_toolkit` for private keys too in [`_load_signer_from_file_prompt()`](https://github.com/repository-service-tuf/repository-service-tuf-cli/blob/3e73d84ce4f86b2ffaeae9b35c4da62c57698432/repository_service_tuf/cli/admin/helpers.py#L216-L218)?

**Review by kairoaraujo — APPROVED** [review]

**Review by kairoaraujo — APPROVED** [review]

**Pull request merged**